src/App.tsx

```diff
@@ -15,48 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  empty = '',
+  alpha = 'alpha',
+  length = 'length',
+}
+
+interface SortVer {
+  sortField: SortType,
+  reverseField: boolean,
+}
+
+function gerPreparedGoods(
+  goods: string[],
```

Comment:
There's a typo in the function name, it should be 'getPreparedGoods' instead of 'gerPreparedGoods'.